kong/cmd/utils/prefix_handler.lua

```diff
@@ -113,7 +113,8 @@ local function gen_default_ssl_cert(kong_config, target)
     end
 
     if not exists(ssl_cert) and not exists(ssl_cert_key) then
-      log.verbose("generating %s SSL certificate (", ssl_cert, ") and key (", ssl_cert_key, ") for ", target or "proxy", " listener")
+      log.verbose("generating %s SSL certificate (%s) and key (%s) for %s",
+        target or "proxy", ssl_cert, ssl_cert_key, "listener")
```

Comment:
```suggestion
        target or "proxy", ssl_cert, ssl_cert_key, "listener")
```

Why do we use the hardcoded `"listener"` in here? Should it be written in the format string instead?